flutter-idea/src/io/flutter/view/EmbeddedJcefBrowser.java

```diff
@@ -33,6 +33,11 @@ public void close() {
 
   }
 
+  @Override
+  public void matchIdeZoom() {
+
```

Comment:
Is this intentionally empty?

Reply:
Chatted offline, this is an experimental browser, does not need implementation here.